src/contract.rs

```diff
@@ -100,7 +100,7 @@ mod tests {
 
     #[test]
     fn increment() {
-        let mut deps = mock_dependencies(&coins(2, "token"));
+        let mut deps = mock_dependencies();
```

Comment:
Here don't just remove them. There is `mock_dependencies_with_balance(&coins(2, "token"))` that should have the same effect.